.gitignore

```diff
@@ -10,3 +10,4 @@
 
 # rspec failure tracking
 .rspec_status
+.DS_Store
```

Comment:
not everyone has this in their global gitignore :), this is a open source one, not for f3d dev :)